modules/ossm-operator-install.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-operator-installation_{context}"]
+= Installing the Operators
+
+The {ProductShortName} installation process introduces an Operator to manage the installation of the control plane within the `istio-operator` namespace. This Operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
+
+Starting with {ProductVersion} {ProductVersion}, you must install the Jaeger Operator and the Kiali Operator before the {ProductName} Operator can install the control plane.
+
+
+[id="ossm-operator-install-jaeger_{context}"]
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following commands to install the Jaeger Operator:
++
+----
+$ oc new-project observability # create the project for the jaeger operator
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/service_account.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role_binding.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/operator.yaml
+----
+
+[id="ossm-operator-install-kiali_{context}"]
+== Installing the Kiali Operator
+You must install the Kiali Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+
+* Review the link:https://www.kiali.io/documentation/getting-started[Kiali] documentation.
+
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following command to install the Kiali Operator:
++
+----
+$ bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali false
+----
+
+
+[id="ossm-operator-install-istio_{context}"]
+== Installing the {ProductName} Operator
+
+.Prerequisites
+
+* Review the link:https://github.com/Maistra/istio-operator/tree/maistra-0.12.0/deploy[Operator templates on GitHub].
```

Comment:
This is a dead link; the branch name is `maistra-0.12`, apparently

Reply:
Yeah. maistra-0.12 is the correct branch.

Reply:
Fixed.

Reply:
Fixed.